css/main.css

```
@@ -705,10 +705,6 @@ footer .top-footer .follow .social-icons .fab, footer .top-footer .follow .socia
   margin-right: .1rem;
 }
 
```

Removing this CSS causes the youtube icon to become too narrow. I suggest limiting the CSS to that class rather than removing it.

```suggestion
footer .top-footer .follow .social-icons .fa-caret-right {
  padding: .5rem 1rem;
}
```

---

Now that #46 is in this doesn't matter.